bundles/org.openhab.binding.ecoflow/README.md

```diff
@@ -0,0 +1,158 @@
+# EcoFlow Binding
+
+This binding provides integration for power stations and micro inverters made by EcoFlow (<https://www.ecoflow.com/>).
+It discovers devices and communicates to them by using the cloud services provided by EcoFLow.
+
+## Supported Things
+
+- EcoFlow cloud API (`ecoflowapi`)
+- Delta 2 power station (`delta2`)
+- Delta 2 Max power station (`delta2max`)
+- PowerStream micro inverter (`powerstream`)
+
+## Discovery
+
+At first, you need to manually create the bridge thing for the cloud API.
+Once that is done, you can initiate a device scan and supported devices will be automatically discovered and added to the inbox.
+
+## Thing Configuration
+
+For the cloud API thing, you need to register for a developer account at EcoFlow's [developer page](https://developer.ecoflow.com).
```

Comment:
Would this make it more clear? Also note the `lower-case-hypen` applies to the id
```suggestion
For the `ecoflowapi` Bridge, you need to register for a developer account at EcoFlow's [developer page](https://developer.ecoflow.com).
```

Reply:
I added the bridge type here, just worded slightly differently.